### PR TITLE
[buteo-sync-plugin-webcal] Use lcWebCal logging category instead of Buteo LOG_* macros. JB#53712

### PR DESCRIPTION
**Pull request by blammit**

### CODE REVIEW

**Review by dcaliste — CHANGES_REQUESTED** [review]
Like in the CalDAV plugin, I put also a line like `qputenv("MSYNCD_LOGGING_LEVEL", "8");` in the tests. It needs to be updated.

Thank you for the changes here to use Qt logging categories.

File: src/webcalclient.h [hunk 1]
```diff
@@ -99,5 +100,6 @@ class WebCalClientLoader : public Buteo::SyncPluginLoader
                                             Buteo::PluginCbInterface* cbInterface) override;
 };
 
+Q_DECLARE_LOGGING_CATEGORY(lcWebCal)
```
Comment (pvuorela) [inline src/webcalclient.h]: Since only one .cpp uses logging, suppose this is not absolutely needed.

**Review by pvuorela — APPROVED** [review]
Seems simple and good, and suppose could be merged already

**Comment by pvuorela** [comment]
Oh, didn't notice Damien's comment on the code tab. Removal enough maybe?

**Comment by dcaliste** [comment]
> Removal enough maybe?

Yes, indeed, one can completely remove the line making the `qsetenv`. After all, I put it there the first time in CalDAV to actually don't have to remember it... While, now that we're using standard Qt logging, one can turn on or off the debug messages with a standard way.

**Review by chriadam — APPROVED** [review]

**Pull request merged**